src/tests/index.test.js

```diff
@@ -0,0 +1,617 @@
+import chai from 'chai';
+import spies from 'chai-spies';
+import Evolv, { INITIALIZED } from '../index.js';
```

Comment:
INITIALIZED is not exported, so tests won't run

Reply:
Fixed these.